spec/percy/cli/snapshot_runner_spec.rb

```diff
@@ -99,8 +99,29 @@
       expect(paths).to match_array(expected_results)
     end
 
+    it 'returns only the files matching the snapshots_regex' do
+      opts = { snapshots_regex: 'xml' }
```

Comment:
It occurred to me that including xml files is kinda odd, as the intention is to grab html files inside a folder - but maybe this is okay to test that we can handle oddball file names.

Reply:
I just picked a file extension that I could pickup with `snapshot_regex` that fit in the web world.

Actually, what's the intention behind having a `snapshot_regex` option? I can include better test cases to reflect the intended usage. It looks like this option extends way back to the very beginning of this CLI.

Reply:
It does the opposite of ignore_regex - specify the files (usually html files) to include in the set of snapshots.  https://github.com/percy/percy-cli/blob/c14fab5a935ea1097329bed4987a23f304420299/lib/percy/cli.rb#L48

I just noticed these two regexes aren't in the docs.  That's a small oversight..

Reply:
Ah ok. I'm going to leave the xml files in the test then 😄